v4/source/users.yaml

```diff
@@ -1556,3 +1557,33 @@
           $ref: '#/responses/Unauthorized'
         '403':
           $ref: '#/responses/Forbidden'
+    get:
+      tags:
+      - users
+      - terms of service
+      summary: Fetches user's latest terms of service action if the latest action for for acceptance.
```

Comment:
That feels a bit clearer to me now. Just a little typo to fix now (and below in the description). `for for acceptance` should probably be `was for acceptance`?

Reply:
Yes, that's a typo. Fixed now.